bee-network/bee-gossip/src/peer/list.rs

```diff
@@ -475,6 +498,12 @@ pub enum PeerState {
     Connected(GossipSender),
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct PeerMetrics {
+    pub(crate) num_dials: usize,
+    pub(crate) identified: Option<u64>,
```

Comment:
It's not obvious to me what this actually represent. Given the name I would expect a boolean.

Reply:
Since it's an `Option` it's kinda like a boolean: `None` ~> not identified, `Some(timestamp)` ~> "identified at".

Reply:
`identified_at` might be a better name, usually `Option`s seem to be named as if they were `Some(name)`.

Reply:
Alright, then. Makes sense Adam.

Reply:
Done